blocks/rich-text/format-toolbar/index.js

```diff
@@ -184,7 +184,7 @@ class FormatToolbar extends Component {
 
 				{ ( isAddingLink || isEditingLink || formats.link ) && (
 					<Fill name="RichText.Siblings">
-						<div style={ { position: 'absolute', ...focusPosition } }>
+						<div style={ { position: 'absolute', transform: 'translateX( -50% )', ...focusPosition } }>
```

Comment:
One sneaky `position` was OK, but now that there are a few properties I think we ought to give this div a `className` and add a new rule to `format-toolbar/style.scss`:

```scss
.blocks-format-toolbar__link-container {
    position: absolute;
    transform: translateX( -50% );
}
```